pkg/types/installconfig.go

```diff
@@ -199,6 +199,7 @@ type InstallConfig struct {
 	// AzureStack: "Manual"
 	// GCP: "Mint", "Passthrough", "Manual"
 	// IBMCloud: "Manual"
+	// OpenStack: "Passthrough"
```

Comment:
Updating field description will requires regenerating the installconfig CRD (i.e. `data/data/install.openshift.io_installconfigs.yaml`). You can run:

```bash
go generate ./pkg/types/installconfig.go
```

The CRD is used for command `openshift-install explain`.